ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -71,20 +73,34 @@
         for (var i = 0; i < deserializedData.visualization.length-1; i++) {
             materials.push(materials1);
         }
-        var points = [], lines = [];
-        var geometry, line;
+        var lineLoopPoints = [], lineSegmentPoints = [], lineSegments = [], objects = [];
+        var geometry, geometries;
         for (var i = 0; i < layerCoordinates.length; i++) {
-            for (const l of layerCoordinates[i]) {
-                points = [];
-                for (var j = 0; j < l.length; j+=3) {
-                    points.push( new THREE.Vector3( l[j], l[j+1], l[j+2] ) );
+            lineSegments = [];
+            for (const line of layerCoordinates[i]) {
+                lineSegmentPoints = [];
+                lineLoopPoints = [];
+                if (line.length === TWOPOINTS) {
+                    for (var j = 0; j < line.length; j+=3) {
+                        lineSegmentPoints.push( new THREE.Vector3( line[j], line[j+1], line[j+2] ) );
+                    }
```

Comment:
this block looks like it's duplicated, can we move it outside the length === TWOPOINTS check?